lib/shopify_app/sessions.rb

```diff
@@ -1,5 +1,5 @@
 module ShopifyApp
-  module SessionsController
```

Comment:
Do we have a standard way of naming concerns? I just saw `LoginProtection` so I guess not